lib/skate/location_search/aws_location_request.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Skate.LocationSearch.AwsLocationRequest do
+  alias Skate.LocationSearch.SearchResult
+
+  @base_arguments %{
+    FilterBBox: [-71.55, 42.05, -70.6, 42.65]
+  }
+
+  @spec search(String.t()) :: {:ok, map()} | {:error, term()}
+  def search(text) do
```

Comment:
question(non-blocking): Why not have return the result of `parse_search_response` when the search request is successful?

Reply:
After mulling this over I decided to refactor it as suggested and I'm pretty happy with the end result, let me know what you think. Commit 08c80418.

Reply:
Nice, looks good to me!